src/base/lockfree_transaction_table.hpp

```diff
@@ -57,15 +60,7 @@ namespace lockfree
   {
     static const id INVALID_TRANID = std::numeric_limits<id>::max ();
```

Comment:
should this moved to lockfree_transaction_descriptor.hpp ?

Reply:
moved